python_modules/dagster/dagster/_core/execution/backfill.py

```diff
@@ -38,6 +38,13 @@ def from_graphql_input(graphql_str):
         return BulkActionStatus(graphql_str)
 
 
+BULK_ACTION_COMPLETED_STATUSES = [
```

Comment:
maybe rename this to `BULK_ACTION_TERMINAL_STATUSES` to avoid confusion with the COMPLETED status